### PR TITLE
Quiz UI result overview

### LINKED ISSUE #977: Modify database functions to show the results in an overview
- [ ] Create a results view, where you can click on a result to see more info about te options and chosen option.
- [ ] Modify the database functions so the right info is supplied to the results view.

### PR DESCRIPTION
**Pull request by LterHaar**
Quiz Result Overview Page

**Description**

Gives an overview of the results of the quiz for each question and visualizes if the question was answered correct and the number of attempts are shown, fixes #977 

**Fixes < issue or discussion number >**

Replaces the original  endquiz results screen.

**How to test**
End Screen - Achievement
End screen

result screen1

### CODE REVIEW

**Comment by Felienne** [comment]
Thanks @LotusterHaar! Ik zie een paar conflicten maar dit durf ik zelf niet op te lossen. Kan jij er naar kijken? Anders vragen we het @TiBiBa volgende week (hij is nu op vakantie helaas)

**Comment by Felienne** [comment]
@TiBiBa maybe you can also have a peek at the conflicts here?

File: templates/quiz-result-overview.html
Comment (TiBiBa) [inline templates/quiz-result-overview.html:117]: In this case we do use the `hedy_link` function. Would be nice to do this consistent, either everywhere on not at all.

File: website/quiz.py
Comment (TiBiBa) [inline website/quiz.py:165]: Can this print be removed before merging?

File: website/quiz.py
Comment (TiBiBa) [inline website/quiz.py:309]: Same for this print

File: website/quiz.py
Comment (TiBiBa) [inline website/quiz.py:310]: Can't we iterate the questions itself instead of using a loop counter. So simply `for question in questions`.

**Review by TiBiBa — DISMISSED** [review]
Hi @LterHaar! I just left some comments, can you take a look? Just some small questions and requests. I only looked at the code and didn't actually test it (yet). Will do that later today and let you know if I have some questions. Otherwise the code looks great, nice!

**Comment by TiBiBa** [comment]
I just fixed the merging errors and it all seems to work fine, awesome! @Felienne I think it would be best to merge this first before #2136 as we do some clean-up to improve the YAML reading structure there. It will be easier to solve the merging conflicts on that PR than the other way around.

**Review by Felienne — CHANGES_REQUESTED** [review]
Looking good indeed, thanks @LterHaar!

A few tiny things (fine if @TiBiBa picks these up)

The overview looks goed, but it seems we lost syntax highlighting in the quiz? 

image

Also, probably very tiny but the overview (goed, pogingen) is in Duch even though my lang is English:

image

And if we are on the topic of languages anyway, this "Good" is a bit Dunglish, "correct" would be nicer:

image

**Comment by TiBiBa** [comment]
> Looking good indeed, thanks @LterHaar!
> 
> 
> 
> A few tiny things (fine if @TiBiBa picks these up)
> 
> 
> 
> The overview looks goed, but it seems we lost syntax highlighting in the quiz? 
> 
> 
> 
> image
> 
> 
> 
> Also, probably very tiny but the overview (goed, pogingen) is in Duch even though my lang is English:
> 
> 
> 
> image
> 
> 
> 
> And if we are on the topic of languages anyway, this "Good" is a bit Dunglish, "correct" would be nicer:
> 
> 
> 
> image
> 
> 

Quiz syntax highlighting is not merged yet and an open PR #2208.

**Comment by Felienne** [comment]
> Quiz syntax highlighting is not merged yet and an open PR #2208.

My bad, I approved it but did not merge yet, merged it now and all looks fine for this PR too.

**Review by Felienne — CHANGES_REQUESTED** [review]
This can be merged when we have cleared up the Dutch/English text issue (@TiBiBa, can you look into that?)

**Comment by TiBiBa** [comment]
I have some issues with fixing the questions iterating. Will leave this a draft for now and fix this on Sunday!

**Comment by TiBiBa** [comment]
@Felienne I think this one is good to go! I made some small fixes:

- Implemented language dependent strings
- Added syntax highlighting to results page
- Fix some small styling issues

**Comment by Felienne** [comment]
> @Felienne I think this one is good to go! I made some small fixes:
> 
> * Implemented language dependent strings
> * Added syntax highlighting to results page
> * Fix some small styling issues

Great! I will try to have a look today but I am lecturing all day so maybe it'll be tomorrow

File: coursedata/texts/nl.yaml
Comment (Felienne) [inline coursedata/texts/nl.yaml:33]:
```suggestion
    correct: "Goed"
```

Comment (Felienne) [reply coursedata/texts/nl.yaml:33]: Vind ik toch wat Nederlandser

File: coursedata/texts/nl.yaml
Comment (Felienne) [inline coursedata/texts/nl.yaml:34]:
```suggestion
    incorrect: "Niet goed!"
```

**Review by Felienne — APPROVED** [review]

**Review by Felienne — APPROVED** [review]
Helemaal in orde nu @LotusterHaar!

**Pull request merged**
